yeonnj/level1/예산.js

```diff
@@ -0,0 +1,15 @@
+function solution(d, budget) {
+  const sortD = d.sort((a, b) => a - b);
```

Comment:
```suggestion
  const sortedList = [...d].sort((a, b) => a - b);
```
인자로 참조형을 받고 사이드 이펙트를 발생하는 함수를 사용 할 때는 한번 복사하고 하는게 좋습니다 🤗 (사이드 이펙트 방지)
+ sort와 같은 동사는 함수이기 때문에 값 변수에는 적절하지 않습니다! 명사형으로 작성하는 것을 권장합니다 :)

Reply:
제가 원본배열을 바꾸는 실수를 했네요,,! 넵 sortedD나 sortedList처럼 명사형으로 작성해주는게 좋은것같습니다 :)